docs/design/2021-06-27-restructure-tests.md

```diff
@@ -0,0 +1,135 @@
+# Proposal: Restructure TiDB Tests
+
+- Author(s): [tison](http://github.com/tison)
```

Comment:
This should be `https://github.com/TisonKun`

Reply:
/cc @tisonkun

Reply:
Oops. Let me create a followup.

Reply:
FYI https://github.com/pingcap/tidb/pull/26517